### PR TITLE
enhance indentation

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

**Comment by Youssef1313** [comment]
I think, string interpolation may be used. Also use `var` in lines 97, 100

**Comment by BillWagner** [comment]
@mairaw Were you going to merge this?

**Comment by mairaw** [comment]
Sorry @BillWagner, spaced out on this one. Yes, I can add string interpolation too.

File: snippets/csharp/VS_Snippets_Remoting/SocketPropertyTester/CS/class1.cs [hunk 1]
```diff
@@ -9,25 +9,25 @@ class SocketPropertyTester
         //<Snippet1>
 		static void ConfigureTcpSocket(Socket tcpSocket)
```
Comment (NextTurn) [inline snippets/csharp/VS_Snippets_Remoting/SocketPropertyTester/CS/class1.cs]:
```suggestion
        static void ConfigureTcpSocket(Socket tcpSocket)
```

Comment (NextTurn) [inline snippets/csharp/VS_Snippets_Remoting/SocketPropertyTester/CS/class1.cs]:
```suggestion
        static void ConfigureTcpSocket(Socket tcpSocket)
```

Comment (mairaw) [reply snippets/csharp/VS_Snippets_Remoting/SocketPropertyTester/CS/class1.cs]: Thanks @NextTurn. Good catch!

**Pull request merged**